### ISSUE TITLE
Protecting IP

### ISSUE THREAD

**Issue by matthewpersico** [nceder/training#1]
You should advise people who are going to fork the repo to make that repo PRIVATE. Otherwise your action of making it private isn't going to stop the IP leak via google search.

In fact, you should ask the powers that be here if you could load the repo here at BBGitHub instead of on GitHub itself.

**Comment by matthewpersico** [comment]
Failing the permission to use BBGitHub, suggest adding something like the following to README.md

If you are going to fork this repo for your future reference, please goto Settings, scroll down to Danger Zone and mark the repo Private. I will be making this repo itself private on Tuesday in order to protect the IP from being Google Indexed; please help me by privatizing your copies. Private repos now cost $0.00 after Microsoft's acquisition of GitHub, Thank you.

**Comment by nceder** [comment]
Good idea! Thanks for the suggestion!

**Comment by jimmythefung** [comment]
Good idea. I tried making my fork private but the setting complains that I am not allowed to make public fork private. The user will have to make a duplicate first and then privatize.
Alternative route is going bbgithub.
Neither methods are friction-less...

**Comment by matthewpersico** [comment]
If @nceder is going to make the original private, then @jimmythefung is right - the correct actions are clone down, push back up into your own org, not as a fork, then privatize. Or just keep the local clone and don't GitHub your copy at all.
